index.html

```diff
@@ -11,48 +11,58 @@
 
 <body>
     <header id="header">
-        <a href="/" id="welcome">Welcome</a>
+        <a href="/" id="welcome" class="only-mobile">Welcome</a>
         <nav>
-            <button>
+            <ul class="only-desktop desktop-menu">
+                <li><a href="#landing">Portfolio</a></li>
+                <li><a href="#about-me">About</a></li>
+                <li><a href="#form-start">Contact</a></li>
+            </ul>
+            <button class="only-mobile">
                 <svg xmlns="http://www.w3.org/2000/svg" class="icon-md" fill="none" viewBox="0 0 24 24"
                     stroke="currentColor">
                     <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
                 </svg>
             </button>
         </nav>
     </header>
-    <main class="landing">
-        <div class="landiv">
-            <h1>Hey there. I'm Abel<br><span class="greeny">I'm a software developer</span></h1>
-            <p id="main-support-text"> I can help you build a product , feature or website Look through some of my work
-                and experience! If you like what you see and have a project you need coded,
-                don’t hestiate to contact me.</p>
-        </div>
-        <div class="logo-cloud">
-            <ul>
-                <li>
-                    <a href="#"><img src="./assets/svg/twitter.svg" alt="twitter"></a>
-                </li>
-                <li>
-                    <a href="#"><img src="./assets/svg/Linkedin-icon.svg" alt="LinkedIn"></a>
-                </li>
-                <li>
-                    <a href="#"><img src="./assets/svg/medium.svg" alt="Medium"></a>
-                </li>
-                <li>
-                    <a href="#"><img src="./assets/svg/github.svg" alt="Github"></a>
-                </li>
-                <li>
-                    <a href="#"><img src="./assets/svg/angellist.svg" alt="Angellist"></a>
-                </li>
-            </ul>
-        </div>
-        <div class="scroll-down">
-            <button><img src="./assets/png/arrowtip.png" alt=""> </button>
-        </div>
+    <main >
+        <section id="landing">
 
-        <section id="my-work">
-            <div class="first section">
+            <div class="landiv">
+                <h1>Hey there. I'm Abel<br><span class="greeny">I'm a software developer</span></h1>
+                <p id="main-support-text"> I can help you build a product , feature or website Look through some of my work
```

Comment:
- [ OPTIONAL] Use the `font-size` and `font-family` values same as the Figma template.